packages/ts/src/components/treemap/index.ts

```diff
@@ -201,17 +201,23 @@ export class Treemap<Datum> extends ComponentCore<Datum[], TreemapConfigInterfac
     const rx = (d: TreemapNode<Datum>): number =>
```

Comment:
Maybe we should rename this to something else, since rx and ry will share the same value. Otherwise in the code it seems a little confusing 
```
      .attr('rx', rx)
      .attr('ry', rx)
```